.vscode/settings.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
should this be checked in? genuinely unfamiliar with conventions for this team/repo so this is a "I am not sure, teach me" question not a "I am skeptical" one ;-p

Reply:
I actually have a similar question. I was getting vscode plugin conflicts on all the imports and this seemed to solve it. I thought it may help but others but if this was just me I probably should ignore the settings from getting checked in.